packages/@aws-cdk/aws-ecs/lib/log-drivers/splunk-log-driver.ts

```diff
@@ -25,9 +25,16 @@ export enum SplunkLogFormat {
 export interface SplunkLogDriverProps extends BaseLogDriverProps {
   /**
    * Splunk HTTP Event Collector token.
+   * @deprecated Use {@link SplunkLogDriverProps.secretToken} instead.
    */
   readonly token: SecretValue;
```

Comment:
Since you are deprecating this field, you should make it optional.  

(Apologies for the earlier guidance about that being a breaking change. Changing a field of a class from required to optional would be a breaking change. But changing a property in an interface used for input props from required to optional is fine. )